scripts/build/termux_step_cleanup_packages.sh

```diff
@@ -1,11 +1,14 @@
 termux_step_cleanup_packages() {
 	[[ -d "$TERMUX_TOPDIR" ]] || return 0
 
-	# Extract available disk space in GB
-	local AVAILABLE=`df -B $((1024**3)) --output=avail "$TERMUX_TOPDIR" | tail -1`
+	# How much space is considered to be enough
+	local CLEANUP_TRESHOLD="$(( 5 * 1024 ** 3 ))" # 5 GB
```

Comment:
This is binary so Gibibytes.
```suggestion
	local CLEANUP_TRESHOLD="$(( 5 * 1024 ** 3 ))" # 5 GiB
```